src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -69,12 +69,19 @@ Not every language or logging framework is supported yet. The following are our
 * [Python logs in context procedures](/docs/logs/logs-context/configure-logs-context-python) for agent [v7.12.0.176](/docs/release-notes/agent-release-notes/python-release-notes)
 * [Ruby logs in context procedures](/docs/logs/logs-context/configure-logs-context-ruby) for agent [v8.6.0 or higher](/docs/release-notes/agent-release-notes/ruby-release-notes)
 
-<Callout variant="tip">
-  APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
-</Callout>
-
 If your APM agent doesn't support our automatic logs in context solution yet, you can continue to use our [manual logs in context solutions](/docs/logs/logs-context/logs-in-context), and forward your logs via our infrastructure agent or supported third-party forwarder.
 
+## Known limitations [#limitations]
+
+APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+
+Here are some additional known limitations:
+
+* Startup logs are not available until after the agent is loaded.
+* If you are using Kubernetes, be aware that we decorate the logs via instrumentation, not from the Kubernetes API. This is separate and apart from the writing logs out to the filesystem. The logs never touch the host or exist in a place where the API can be called.
+* When an exception is thrown from your application, you may not see all stack traces in the associated logs in context for Java or .NET agents. As a workaround, you can change your drop filter rules.
```

Comment:
In Java, no stack trace will ever be sent when an exception is logged. While the language here covers this case, it may give the impression that some might be sent.

Reply:
Thanks for making that clarification!